pom.xml

```diff
@@ -22,12 +22,18 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
-  <groupId>com.datatorrent</groupId>
-  <artifactId>dt-framework</artifactId>
-  <version>3.3.0-SNAPSHOT</version>
+  <parent>
+    <groupId>org.apache</groupId>
+    <artifactId>apache</artifactId>
+    <version>16</version>
+  </parent>
+
+  <groupId>org.apache.apex</groupId>
+  <artifactId>apex</artifactId>
+  <version>3.3.0-incubating-SNAPSHOT</version>
   <packaging>pom</packaging>
 
-  <name>Apache Apex Core</name>
+  <name>Apache Apex</name>
```

Comment:
3.2 is not consistent with 3.3. +1 to change to "Apache Apex" both in 3.2 and 3.3.